src/SFA.DAS.Commitments.Api.IntegrationTests/Helpers/CommitmentsApi.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
```

Comment:
Could we consider using the existing Commitment.API.Client for these calls? Just in case the Endpoint changes. One less place to consider having to make changes

Reply:
I did consider using the existing client for the calls, but I wanted more fine grained control for better timings, e.g. in the load test, by default, we don't deserialise the body contents.
For subsequent general integration tests, I think using the client is the way to go.